src/components/MapContainer.tsx

```diff
@@ -47,27 +40,15 @@ export const MapContainer = ({ onClick }: { onClick?: __esri.ViewImmediateClickE
       options: {
         view: mapView.current,
         quadWord: import.meta.env.VITE_DISCOVER,
-        baseLayers: [
-          'Hybrid',
-          {
-            label: 'Lite',
-            function: () =>
-              new VectorTileLayer({
-                url: urls.liteVector,
-              }),
-          },
-          'Terrain',
-          'Topo',
-          'Color IR',
-        ],
+        basemaps: ['Lite', 'Hybrid', 'Terrain', 'Topo', 'Color IR'],
```

Comment:
Is there a reason you chose to use the basemap version for atlas? Are there specific use cases for when to choose one or the other? I assumed the non-webmap basemaps were more appropriate for atlas? What am I missing?

Reply:
If we want the labels to show up on top of operational data, then you have to use the basemap version. There is no single tiles service that can be used to break out the labels. I switched all of them to the basemap prop because of the randomize code below.

Reply:
Is the operational data in atlas the address points and land ownership then?